Domains/QuantityTakeoffsAspects/QuantityTakeoffsAspects.ecschema.xml

```diff
@@ -158,13 +174,21 @@
     <ECEntityClass typeName="PileAspect" description="An aspect that defines the depth of a vertical structural element of a deep foundation - pile." >
         <BaseClass>QuantityTakeoffAspect</BaseClass>
         <ECCustomAttributes>
-            <ClassHasHandler xmlns="BisCore.01.00.00"/>
+            <Deprecated xmlns="CoreCustomAttributes.01.00.03">
```

Comment:
for reporting cases, where will "EmbedmentDepth" be stored once we deprecate this aspect?

Reply:
Created an issue for the StructuralPhysical team.